src/app/lib/device/ext-player.js

```diff
@@ -221,11 +221,12 @@
         folderName = path.resolve(folderName);
         win.info('Scanning: ' + folderName);
         var appIndex = -1;
-        var fileStream = readdirp(folderName, {
+        var fileStream = readdirp({
+            root: folderName,
```

Comment:
Please lock the version of readdirp in package.json, because latest version of readdirp (3.4.0) - there are no root option.

Reply:
https://github.com/datnguyencse/popcorn-desktop/blob/development/package.json#L80

https://github.com/datnguyencse/popcorn-desktop/blob/development/yarn.lock#L5574

current use version 2.0.0

Reply:
Ok got it, I install modules by npm, not yarn.
Install modules by yarn cause some error.